transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessengerTransport.kt

```diff
@@ -0,0 +1,34 @@
+package com.genesys.cloud.messenger.transport.core
+
+import com.genesys.cloud.messenger.transport.shyrka.receive.DeploymentConfig
+import com.genesys.cloud.messenger.transport.util.DefaultTokenStore
+import com.genesys.cloud.messenger.transport.util.TokenStore
+
+/**
+ * The entry point to the services provided by the transport SDK.
+ */
+class MessengerTransport(configuration: Configuration, private val tokenStore: TokenStore) {
+
+    constructor(configuration: Configuration, storeKey: String) : this(configuration, DefaultTokenStore(storeKey))
+
+    constructor(configuration: Configuration) : this(configuration, "com.genesys.cloud.messenger")
```

Comment:
I'm unsure why we ever really needed it, frankly. I'm fine getting rid of it.

Reply:
@crumpf do you have an opinion?

Reply:
As I recall, that `tokenStoreKey` is basically used as a namespace for the token that we're trying to secure. But, if it's not necessary to be in the `Configuration` and we can just use our own internally-defined name that seems like it makes setting up the config for our SDK more simple.

Reply:
So... what's the consensus? Do we keep the key customization? I can't think of a use-case where a caller would require it... it seems to serve no real practical purpose unless I'm missing something.

Reply:
As far as I understand on iOS it is possible to get an access to the store from any application if you know the `tokenStoreKey`.  Given that we have an open source repo and the default value is easy to acces, we want our clients to explicitly provide a value for this `tokenStoreKey` to prevent them from using the default. But if I am wrong (@crumpf ) it doesn't really matter.

Reply:
If that's the case it sounds like we shouldn't even _have_ a default value but rather require the caller to explicitly provide it. @crumpf can you confirm Anton's understanding?

Reply:
@bjdupuis not intentionally dragging my feet here, just doing some investigations to make sure things are well-understood in this area

Reply:
I just wanted to do some sleuthing to make sure, but I don't think any other application should be able to access items we put in the keychain. Per docs:

https://developer.apple.com/documentation/security/keychain_services/keychains

> An app can access only its own keychain items, or those shared with a group to which the app belongs.

Even with the mechanisms that exist for using keychain access groups to explicitly share keychain info between apps (https://developer.apple.com/documentation/security/keychain_services/keychain_items/sharing_access_to_keychain_items_among_a_collection_of_apps), that data can only be shared between the apps that are signed with a developer's account certificate.

Reply:
Thanks for looking into it, @crumpf . Given this information, is the consensus that we should remove the key as a settable thing in the constructor and just use the internal key?

Reply:
Given that keychain access does protect other apps to access the storage, and `SharedPrefs` are using MODE_PRIVATE we can get rid of this parameter/constructor. If customer wants to use their own `tokenStoreKey`  they have a freedom to implement their own TokenStore. Otherwise we have a default one.